enginetest/enginetests.go

```diff
@@ -608,7 +619,14 @@ func TestOrderByGroupBy(t *testing.T, harness Harness) {
 			}
 			rowCount++
 			require.NoError(t, err)
-			val := row[0].(int32)
+
+			var val int64
+			if isServerTest {
+				val = row[0].(int64)
+			} else {
+				val = int64(row[0].(int32))
+			}
```

Comment:
When we have to switch on the `isServerTest` behavior, it would be nice to document why and if there's something we should do longer term to clean it up. I think long term, we wanna get to enginetests having the same output as the wire tests, so documenting the differences in code and adding TODOs with next steps on how to resolve the difference would be super helpful.